web/tests/fieldsets-parser-spec.js

```diff
@@ -0,0 +1,123 @@
+describe('FieldsetParserService', function () {
+
+    beforeEach(function () {
+        fixture.setBase('tests/api-description');
+        this.fieldset = fixture.load('forms/pk/get.json').OK.response_data;
+
+        angular.module('formly', []);
```

Comment:
Is this not a module from a library you should be loading, instead of defining yourself?